core/templates/dev/head/pages/exploration_editor/editor_tab/test_interaction_modal_directive.html

```diff
@@ -32,7 +32,7 @@
   <br>
   <div class="submit-answer-btn">
     <md-button class="oppia-learner-confirm-button protractor-test-submit-answer-button"
-               ng-disabled="!interactionAnswerIsValid"
+               ng-disabled="!isSubmitButtonDisabled()"
```

Comment:
From the names, this seems to be exactly the opposite of what it should be. Is that an error?